src/AccountSettings/components/AccountDeletionDialog.tsx

```diff
@@ -136,7 +130,6 @@ function AccountDeletionDialog(props: AccountDeletionDialogProps) {
       )
 
       await props.sendTransaction(transaction)
-      setTimeout(onDelete, 1000)
```

Comment:
Doesn't this change mean that the dialog never gets closed in case of an account merge?

Reply:
It will get closed because `onSubmissionCompleted` which is added on line 280 will call `onDelete` after the merge is done

Reply:
Ahh, of course! I think I confused `props.onDelete` and `onDelete` for a moment.

Good that you cleaned that up! 👍